### ISSUE TITLE
Max amount of homes

### ISSUE THREAD

**Issue by Alex-Thunderhunter** [ShimmyMySherbet/MultipleHomes#1]
Hello, every time a player tries to use /sethome, the plugin says "Sorry, but you have reached the max amount of homes: 0/1". The permissions are working correctly though.

**Comment by LyeZinho** [comment]
i having this problem too but i realy cant understand this code

**Comment by LyeZinho** [comment]
I read the code 

probably the code error is on sethome.cs

```cs
                if (count >= max)
                {
                    UnturnedChat.Say(caller, $"Sorry, but you have reached the max amount of homes: {count}/{max}");
                    return;
                }
```

**Comment by ShimmyMySherbet** [comment]
Thanks for the fix, since this plugin was a quick throw together without any testing, a few of these issues fell through the cracks.

**Comment by ShimmyMySherbet** [comment]
The fix has been released in version 1.1

**Comment by Alex-Thunderhunter** [comment]
It's working flawlessly now, thank you! If anymore issues arise, I'll be sure to open another issue.
